cmd/preflight/cmd/check_container.go

```diff
@@ -6,10 +6,20 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"net/http"
 	"os"
 	"path/filepath"
 	rt "runtime"
 	"strings"
+	"time"
+
+	"github.com/go-logr/logr"
```

Comment:
This should not move like it is. Is the IDE moving this block up? Or is the linter suggesting that this be moved ahead of "local" imports?

The order should be:
stdlib

project

3rd party